static/css/main.css

```diff
@@ -1,27 +1,153 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400&display=swap");
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
   text-decoration: none;
+  font-family: "Roboto", sans-serif;
```

Comment:
It was another font-family we decided. 
https://fonts.google.com/specimen/Poppins

Reply:
My bad. Accidentally took the wrong links from google fonts. I'll change that.